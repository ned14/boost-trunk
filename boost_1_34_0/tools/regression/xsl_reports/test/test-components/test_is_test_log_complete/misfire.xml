<tests>
<test-log library="config" test-name="config_link_test" test-type="run" test-program="libs/config/test/link/main.cpp" target-directory="status/config_link_test.test/borland-5.8.2/debug" toolset="borland-5.8.2" show-run-output="false">
<compile result="succeed" timestamp="xxx">
"c:/progra~1/borland/bds/4.0/bin/bcc32.exe" -j5 -g255 -q -c -P -Ve -Vx -a8 -b- -v -Od -tWC -tWR -tWC -WM- -DBOOST_ALL_NO_LIB=1 -DBOOST_CONFIG_NO_LIB=1 -DBOOST_DYN_LINK=1 -I".." -I"c:/progra~1/borland/bds/4.0/include/" -o"C:\Users\Misha\Stuff\boost\RC_1_34_0\boost\tools\regression\test\test-cases\borland\actual\status\config_link_test.test\borland-5.8.2\debug\main.obj" "g:\boost\rc-1-34\boost\libs\config\test\link\main.cpp"

g:\boost\rc-1-34\boost\libs\config\test\link\main.cpp:
</compile>
<link result="fail" timestamp="xxx">
set "PATH=c:/progra~1/borland/bds/4.0/bin/;%PATH%"
       "c:/progra~1/borland/bds/4.0/bin/bcc32.exe" -v -q -v -v -tWD -tWC -tWR -tWC -WM-  -L"c:/progra~1/borland/bds/4.0/lib" -e"C:\Users\Misha\Stuff\boost\RC_1_34_0\boost\tools\regression\test\test-cases\borland\actual\status\config_link_test.test\borland-5.8.2\debug\config_link_test.exe" @"C:\Users\Misha\Stuff\boost\RC_1_34_0\boost\tools\regression\test\test-cases\borland\actual\status\config_link_test.test\borland-5.8.2\debug\config_link_test.exe.rsp"

</link>
<run result="succeed" timestamp="xxx">
</run>
</test-log>
</tests>