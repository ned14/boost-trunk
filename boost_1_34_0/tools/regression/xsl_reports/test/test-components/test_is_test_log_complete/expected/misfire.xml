<?xml version="1.0" encoding="utf-8"?>
<tests>
<result name="config_link_test" target-directory="status/config_link_test.test/borland-5.8.2/debug" result="false"/>
</tests>
